table.go

```diff
@@ -69,23 +73,44 @@ func (rt *RoutingTable) Update(p peer.ID) (evicted peer.ID, err error) {
 		bucketID = len(rt.Buckets) - 1
 	}
 
+	var full = 0
```

Comment:
something as intensive as this should maybe be enabled with a flag. could just be a module scope `var`